ksql-engine/src/main/java/io/confluent/ksql/structured/SchemaKTable.java

```diff
@@ -192,4 +190,80 @@ public SchemaKGroupedStream groupBy(
         functionRegistry,
         schemaRegistryClient);
   }
+
+  @SuppressWarnings("unchecked")
+  public SchemaKTable join(
```

Comment:
should we add unit tests for these new methods? Same in `SchemaKStream`

Reply:
These have been added.